docs/admin/engines/nosql-engines.rst

```diff
@@ -42,7 +42,7 @@ Extra Dependencies
 ------------------
 
 For using :ref:`engine redis_server` or :ref:`engine mongodb` you need to
-install additional packages in Python's Virtual Environment of your SearxNG
+install additional packages in Python's Virtual Environment of your SearXNG
 instance.  To switch into the environment (:ref:`searx-src`) you can use
```

Comment:
`searx-src` --> `searxng-src`.

Reply:
This is a reference to ..

https://github.com/searxng/searxng/blob/823a2fc7926441ba4375d74340a7ab08ac85ef9d/docs/admin/installation-searx.rst#L37-L40

IMO it is better we do not change anchors, otherwise we have to change all references (and [intersphinx](https://certik.github.io/sphinx/ext/intersphinx.html) links from the outside would be broken).

Reply:
If I understand correctly: 
* It is an internal reference
* It is an "issue" if some external pages make a reference to this anchor (the HTML anchor).

Am I missing something?

Reply:
> It is an internal reference

the _landing point_ is `.. _searx-src:` (see above)

> It is an "issue" if some external pages make a reference to this anchor (the HTML anchor).

Not the HTML anchor.  It is a bit OT since I suppose nobody from outside has linked this _landing point_, but to understand I will give a short introduce to what [intersphinx](https://certik.github.io/sphinx/ext/intersphinx.html) is ...

With intersphinx in your project you generate a `objects.inv` file which is also distributed, e.g here is our https://searxng.github.io/searxng/objects.inv

In this file there is a mapping from the name of the _landing point_ to the final URL.

Other projects can refer this file, like we do with some other (external) projects ...

https://github.com/searxng/searxng/blob/823a2fc7926441ba4375d74340a7ab08ac85ef9d/docs/conf.py#L92-L99

The object.inv files are loaded at build time of the (HTML) documentation ..

```
make docs
...
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://flask.palletsprojects.com/objects.inv...
loading intersphinx inventory from https://jinja.palletsprojects.com/objects.inv...
loading intersphinx inventory from https://return42.github.io/linuxdoc/objects.inv...
loading intersphinx inventory from https://www.sphinx-doc.org/en/master/objects.inv...
```

With the help of this mappings, sphinx can create HTML links to HTML anchors in external external projects.

In our documentation you can find some examples how external pages can be referenced ...

https://searxng.github.io/searxng/dev/reST.html#smart-refs

![grafik](https://user-images.githubusercontent.com/554536/133054662-dfcae43f-60c9-413b-9366-73157adbb775.png)

hope I could clarify ..

Reply:
Yes it clarify, thank you.

I still think we can be break the interlink: I'm nearly sure that no other external documentation makes a link to this reference.
But that's not important.

Reply:
> I'm nearly sure that no other external documentation makes a link to this reference.

:-) .. d'accord

> I still think we can be break the interlink

Yes we could, but in this case I won't, because it refers to https://searxng.github.io/searxng/admin/installation-searx.html#searx-src where the installation of the `/usr/local/searx/searx-src` is described.

I think we will never change this folder like we will never change the name `searx` of the python package.